catalog/solr/catalog-solr-external-provider/src/main/java/ddf/catalog/solr/external/SolrHttpCatalogProvider.java

```diff
@@ -263,21 +267,22 @@ private void updateServer() {
     }
 
     private CatalogProvider getProvider() {
-        if (isServerUp(getServer())) {
-            if (resolver == null) {
-                provider = new SolrCatalogProvider(getServer(),
-                        filterAdapter,
-                        solrFilterDelegateFactory);
-            } else {
-                provider = new SolrCatalogProvider(getServer(),
-                        filterAdapter,
-                        solrFilterDelegateFactory,
-                        resolver);
+        if (firstUse) {
```

Comment:
Not sure why this was added back in. The `firstUse` pattern makes no sense with a fault-tolerant `SolrServerFactory`. Either the itests need to be updated or there's another problem, but I don't think this change is the correct fix. We can discuss more later, since there's not enough context here for why this logic was brought back.